src/Logic.hs

```diff
@@ -1152,9 +1193,12 @@ describeStatus (BaseBranch projectBaseBranchName) prId pr state = case Pr.classi
   PrStatusSpeculativeConflict -> "Failed to speculatively rebase. \
                                  \ I will retry rebasing automatically when the queue clears."
   PrStatusFailedBuild url -> case Pr.unfailedIntegratedPullRequestsBefore pr state of
+    -- TODO: On Fridays the retry command is also `retry on friday`. We
+    --       currently don't have that information here. Is that worth
+    --       including?
```

Comment:
We try to avoid `TODO`s without associated issue numbers. I think this is fine
so long as it's there in the comment. You can remove the `TODO` and the question.